example/grpc/helloworld_client/main.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"log"
+	"os"
+	"time"
+
+	pb "go.opencensus.io/examples/grpc/proto"
+	"go.opencensus.io/plugin/ocgrpc"
+	"go.opencensus.io/stats/view"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"opencensus-go-signalfx/exporter/signalfx"
+)
+
+const (
+	address     = "localhost:50051"
+	defaultName = "world"
+)
+
+func main() {
+	// Register stats and trace exporters to export
+	// the collected data.
+	exporter, err := signalfx.NewExporter(signalfx.Options{Token: "SIGNALFX_TOKEN"})
```

Comment:
Let's read the token from an environment variable so that you can run the example without modification

Reply:
done